ui2/public/favicon.svg

```diff
@@ -0,0 +1,13 @@
+<svg width="41" height="41" viewBox="0 0 41 41" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <style>
+    path {
+      stroke: #3A465F;
+    }
+    @media (prefers-color-scheme: dark) {
+      path {
+        stroke: #4BE725;
+      }
+    }
+</style>
```

Comment:
(This changes the tab favicon color depending on the system color scheme!)

Reply:
nice!